publications/publication-guidelines.md

```diff
@@ -0,0 +1,9 @@
+# CHAOSS Data Science Publication Guidelines
+
+One goal of the CHAOSS Data Science Working Group (WG) is to begin publishing the research and outcomes from our projects so that people can learn from our findings while also raising the visibility of CHAOSS and this WG. Our current list of projects can be found as [Issues labelled with "Project"](https://github.com/chaoss/wg-data-science/issues?q=is%3Aissue%20state%3Aopen%20label%3Aproject) in the Data Science WG repository, and we have an issue template that anyone can use to propose a new project.
+
+Open source program offices, community managers, and other people working in corporate open source environments have been the biggest audiences for CHAOSS tools and metrics, so to make it easier for these people to consume our research, we are focusing on producing reports that are in the style of white papers, like what you can find published by [Linux Foundation Research](https://www.linuxfoundation.org/research). Ideally, we would like work with LF Research to release our reports, since we are an LF project. However, not everything we publish be appropriate for an LF Research report, so alternatively, some will be published as reports on the CHAOSS site. Good reports to model ours after are the LF / Harvard [Census II](https://www.linuxfoundation.org/hubfs/Research%20Reports/lfr_harvard_censusII_mar2022_042824b.pdf?hsLang=en) and [Census III](https://www.linuxfoundation.org/hubfs/LF%20Research/lfr_censusiii_120424a.pdf?hsLang=en) reports, since they are well-researched, but presented in a style that is accessible for a wide audience (note that we expect our reports to be much shorter). 
+
+From a CHAOSS Data Science WG perspective, these reports are not intended to be academic publications; however, the people leading the research for a report are welcome (and encouraged) to submit variations of the research into academic publications. One of the reasons we chose the white paper / LF research report format is because we can quickly get our research published, since the academic publishing pipeline is often measured in years, so it is important to note that we will **not** delay the publishing of our reports, so these academic publications will almost always come after CHAOSS has published our report. We also require that you provide appropriate credit via an acknowledgements section to all of the other people who have contributed to the research and acknowledge the CHAOSS project in any additional publications.
```

Comment:
Change: publishing of our reports, so these academic publications will almost always come after CHAOSS has published our report.
To: publishing of our reports, so academic publications will almost always come after CHAOSS has published a report.